doc/tutorials/setup_logical_partition_scenario_on_nuc.rst

```diff
@@ -0,0 +1,247 @@
+      $ sudo umount /mnt
+      # Mount the Clear Linux OS root filesystem on the USB flash disk
+      $ sudo mount /dev/sdb3 /mnt
+      $ sudo cp -r <path-to-kernel-module-folder-built-in-step1>/lib/modules/* /mnt/lib/modules
+      $ sudo umount /mnt
+
+#. Copy the bootable kernel image to the /boot directory:
+
+   .. code-block:: none
+
+      $ sudo cp <path-to-kernel-image-built-in-step1>/bzImage /boot/
+
+Update ACRN hypervisor Image
+****************************
+
+#. Before building the ACRN hypervisor, you need to figure out the I/O
+   address of the serial port, and the PCI BDF addresses of the SATA controller
+   and the USB controllers on the NUC. Enter the following command to get the
+   I/O addresses of the serial port. The NUC supports one serial port ttyS0.
+   You'll need to connect the serial port to the development workstation, in
+   order to access the ACRN serial console to switch between pre-launched VMs:
+   
+   .. code-block:: none
+
+      $ dmesg | grep ttyS0
+      [    0.000000] console [ttyS0] enabled
+      [    1.562546] 00:01: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is
+      a 16550A
+
+   The following command prints detailed information about all PCI buses and
+   devices in the system:
+
+   .. code-block:: none
+
+      $ sudo lspci -vv
+      00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21) (prog-if 30 [XHCI])
+              Subsystem: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller
+      00:17.0 SATA controller: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode] (rev 21) (prog-if 01 [AHCI 1.0])
+              Subsystem: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode]
+      00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
+              Subsystem: Intel Corporation Ethernet Connection I219-LM
+
+   .. note:: 
+
+      Double check PCI devices BDF defined in the
+      ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``
+      with the information reported by the ``lspci -vv`` command.
+
+#. Clone the ACRN source code and configure the build options
+
+   Please refer :ref:`getting-started-building` to setup ACRN build environment
+   on your development workstation.
+
+   Clone the ACRN source code:
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+      $ cd acrn-hypervisor/hypervisor
+
+   Configure the build options:
+
+   .. code-block:: none
+   
+      $ make defconfig BOARD=nuc7i7bnh
+      $ make menuconfig
+
+   Updates the following configure item:
+
+     *  Set ACRN Scenario as "Logical Partition VMs";
+     *  Set Maximum number of VCPUs per VM as "2";
+     *  Set Maximum number of PCPU as "4";
+     *  Clear/Disable “Enable hypervisor relocation”.
+
+   We recommend keeping the default values of items not mentioned above.
+
+#. Check GRUB boot loader name. 
```

Comment:
I don't understand this step, what is its purpose?

Reply:
@gvancuts  Thanks! On the current design, logical partition depends on the default boot loader (GRUB) and its name of ubuntu 16.04. Ubuntu installing maybe guarantee this, otherwise, the hypervisor will fails to boot. Only double check it here, do you think we can skip this double check?

Reply:
Update double check boot loader name steps.